docs/samples/browser-plugin-meetings/app.js

```diff
@@ -224,6 +227,24 @@ function unregister() {
     });
 }
 
+
+async function getGuestAccessToken() {
+
+  await axios({
+    method: 'post',
+    url: 'https://2j1g168nf8.execute-api.us-east-1.amazonaws.com/dev/guest',
+    data: {
+      name: guestName.value
+    }
+  }).then(function (response) {
+    console.log(response);
```

Comment:
would be better for the log to have some meaningful message like "guest token response:"

Reply:
done